src/components/DataDuesAction/index.js

```diff
@@ -13,10 +13,9 @@ import {
   unknown,
   validationSchema
 } from './schema'
-import './styles.scss'
 
 const DataDuesHeader = () => (
-  <>
+  <div id="data-dues-page" className="data-dues-page">
```

Comment:
@duranmla this is the data dues component, the page should be in the page folder. `DataDuesHeader` only contains the text before the form. Let's remove this